src/main/scala/dpla/ingestion3/enrichments/StringUtils.scala

```diff
@@ -161,5 +162,56 @@ object StringUtils {
       *   whitespace (\s)
       */
     private def beginAndEndPunctuationToRemove = """[;:/,-\\t\\r\\n\s]"""
+
+    /**
+      * Removes matching leading and trailing brackets (square, round and curly braces)
+      * from a string
+      */
+    val stripBrackets: SingleStringEnrichment = {
+      val replacemnetRegex = for( (p, r) <- bracketPatterns if value.matches(p)) yield r
```

Comment:
I think you meant to spell this `replacementRegex`.  Looks like the mis-spelling is consistent and not causing any bugs, so no biggie.